.pkgr.yml

```diff
@@ -8,3 +8,4 @@ before:
 after_install: "contrib/scripts/after_install.sh"
 targets:
   ubuntu-16.04:
+  ubuntu-18.04:
```

Comment:
Although most programs don't mind, I think the standard is to have a newline at the end of a file.